app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -153,9 +161,9 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         MyApp.LogDebug(LOG_TAG, "screen width = " + screenWidth);
         MyApp.LogDebug(LOG_TAG, "time width " + getResources().getDimension(R.dimen.time_width));
         screenWidth -= getResources().getDimension(R.dimen.time_width);
-        int maxCols = HorizontalSnapScrollView.calcMaxCols(getResources(), screenWidth);
-        MyApp.LogDebug(LOG_TAG, "max cols: " + maxCols);
-        columnWidth = (int) ((float) screenWidth / maxCols); // Width for the row column
+        maxRoomColumnsVisible = HorizontalSnapScrollView.calcMaxCols(getResources(), screenWidth);
+        MyApp.LogDebug(LOG_TAG, "max cols: " + maxRoomColumnsVisible);
+        roomColumnWidth = (int) ((float) screenWidth / maxRoomColumnsVisible); // Width for the room column
```

Comment:
:lipstick: I think the variable name is self-explanatory enough so we don't need the comment anymore.